### ISSUE TITLE
The generated source is included two times in IntelliJ Idea 

### ISSUE THREAD

**Issue by angelcervera** [trueaccord/ScalaPB#126]
## Description
After import a sbt project using ScalaPB plugin, there are two source code folders included in `Project Settings -> Modules`:

- target/scala-2.11/src_managed/main
- target/scala-2.11/src_managed/main/compiled_protobuf

## Workaround
To work properly, it is necessary **remove the first one**.

## Steps

1. Checkout https://github.com/angelcervera/pbf4scala
2. generate code executing `$ sbt compile`
3. Open Idea and import the project
4. `Build -> Rebuild All` throws error: `Blob is already defined ......`
5. `Project Structure -> Modules`: Remove `target/scala-2.11/src_managed/main` from `Source Folders`
6. `Build -> Rebuild` Works correctly.

## Environment:

- IntelliJ Idea version 15.0.6
- Sbt version 0.13
- ScalaPB version 0.4.21

**Comment by seigert** [comment]
Well, seems like with `scalapb:1.0.3`, `sbt:1.4.7` and `IntelliJ IDEA:21.1` this problem resurrected: after `build.sbt` improt there is once again two `generate sources root` present:
- target/scala-2.13/src_managed/main
- target/scala-2.11/src_managed/main/scalapb

Sadly, there is no longer `PB.protobufConfig` option.

**Comment by thesamet** [comment]
Hi @seigert , this is considered an IntelliJ Scala plugin bug since the project compiles in SBT. It would be best to report it to the IntelliJ Scala plugin team. In the meantime, you can try to update build.sbt to generate into sourceManaged.value directly rather than to scalapb subdirectory. This would make ScalaPB's source tree coincide with the default one:

```
Compile / PB.targets := Seq(
  scalapb.gen() -> (Compile / sourceManaged).value
)
```

Don't forget to `sbt clean` and reimport into intellij

**Comment by rdesgroppes** [comment]
Did someone file a IntelliJ Scala plugin bug?

**Comment by thesamet** [comment]
@rdesgroppes This problem is fixed in IntelliJ 2021.1.1 and sbt-protoc 1.0.4.

**Comment by rdesgroppes** [comment]
Thanks, @thesamet! I happened to be stucked on an older version:
```
intellij-idea-ultimate  2020.3.4                    293    2020.3/stable    jetbrains✓    classic
```
... sorry for that!
